timers.js

```diff
@@ -1,4 +1,27 @@
 
+$(document).ready(function () {
 
+  seconds = 0;
+  timerID = "";
 
+  $("#start").on("click",doStart);
```

Comment:
I’d set something up that would prevent it from responding if licked a second time, by creating a new variable, here called clickOnce, initializing it to zero, then changing the event handler to only run if is 0 and change its value to 1 the first time it gets clicked:


    if (clickOnce < 1) {
      $("#start").on("click",doStart);
      clickOnce = 1;
    }


From:  Adrian Maseda <notifications@github.com>
Reply-To:  ga-wdi-exercises/timer_js <reply@reply.github.com>
Date:  Monday, June 6, 2016 at 1:58 PM
To:  ga-wdi-exercises/timer_js <timer_js@noreply.github.com>
Cc:  Kevin Golden <kwg@cs.net>, Author <author@noreply.github.com>
Subject:  Re: [ga-wdi-exercises/timer_js] It works (#136)


In timers.js:
>  
> +  $("#start").on("click",doStart);
Right now if I click on the timer 10 times, it will start 10 different timers (i.e., the timer on-screen will increment more than one number per second). How would you go about preventing this from happening?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.